aiortc/contrib/media.py

```diff
@@ -336,6 +336,11 @@ def addTrack(self, track):
             if self.__container.format.name == "image2":
                 stream = self.__container.add_stream("png", rate=30)
                 stream.pix_fmt = "rgb24"
+            elif self.__container.format.name == "v4l2":
+                # rate=31 workaround for error:
```

Comment:
This framerate looks specific to *your* device?

Reply:
I suspected the problem is in the ffmpeg library. I couldn't figure out where else to fix it. Any setting of <= 30 fps produces the dts failure, where the video will freeze when it gets to around dts=~1200. By all means, if you can fix it elsewhere, I will be happy to test it in my setup.

The raw format seems to be the only one that works for writing to a device (e.g. /dev/video2 v4l2loopbak), which necessitated supplying a new name to handle it. The else case does not work. Otherwise, the file writes work fine with the else case, as before.
--shaun